### PR TITLE
fix: folder mapping deprecation warning (#133)

### PR DESCRIPTION
**Pull request by coderaiser**
Fixed deprecation warning:
image

### CODE REVIEW

**Comment by varwasabi** [comment]
Can confirm that this change fixes the message from showing.

**Review by ljharb — CHANGES_REQUESTED** [review]
This is a breaking change; it will break consumers like https://github.com/es-shims/Array.prototype.at/blob/main/index.mjs#L2, and any users on node 12.17 (or any other node that understands "exports" but not subpath patterns)

**Comment by coderaiser** [comment]
Updated to get `node v12` and `node v16` supported according to workaround https://github.com/nodejs/node/issues/38353#issuecomment-825221607

File: package.json [hunk 1]
```diff
@@ -31,7 +31,16 @@
...
+		"./": "./",
+		"./2020/*": "./2020/*.js",
+		"./2019/*": "./2019/*.js",
+		"./2018/*": "./2018/*.js",
+		"./2017/*": "./2017/*.js",
+		"./2016/*": "./2016/*.js",
+		"./2015/*": "./2015/*.js",
+		"./helpers/*": "./helpers/*.js",
+		"./5/*": "./5/*.js",
+		"./*": "./*.js"
```
Comment (ljharb) [inline package.json:43]: unfortunately, this causes ` Cannot find module '$PWD/node_modules/es-abstract/2020/RequireObjectCoercible.js.js'` or similar in node 16.6, 12.23, and 12.17 when I try to import with the extension - when i try to import without the extension, I get `Cannot find module '$PWD/node_modules/es-abstract/2020/RequireObjectCoercible'` in node v12.17 - so this doesn't suffice.

If you remove the `.js` from all the RHS's of these lines, then ESM works, but CJS (the vastly more important consumer for the foreseeable future) is no longer able to require them without extensions.

**Comment by coderaiser** [comment]
Closed due to removed `exports`.

**Pull request closed without merging**